lib/galaxy/util/__init__.py

```diff
@@ -1015,7 +1015,7 @@ def roundify(amount, sfs=2):
         return amount[0:sfs] + '0' * (len(amount) - sfs)
 
 
-def unicodify(value, encoding=DEFAULT_ENCODING, error='replace', strip_null=False):
+def unicodify(value, encoding=DEFAULT_ENCODING, error='replace', strip_null=False, log_exception=True):
```

Comment:
It seems some files were not added to the PR? The new argument is not used in any unicodify call.

Reply:
ah, thanks!